resources/js/lang/en/rooms.js

```diff
@@ -162,7 +164,8 @@ export default {
         disabled: 'Disabled',
         enabled: 'Enabled',
         onlyForGuestsEnabled: 'Enabled only for guests'
-      }
+      },
+      recordAttendance: 'Log attendance'
```

Comment:
Maybe `Log attendance of participants`? Since it is not only the logs for the current user!